src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/features/form/layout/fetch/fetchFormLayoutSagas.ts

```diff
@@ -90,3 +97,64 @@ export function* watchFetchFormLayoutSettingsSaga(): SagaIterator {
   ]);
   yield call(fetchFormLayoutSettingsSaga);
 }
+
+export function* updateCurrentViewSaga({ newView, runValidations }: IUpdateCurrentView): SagaIterator {
```

Comment:
Not sure this is the right file for this saga? Should it be in the `updateFormLayoutSagas`?

Reply:
You're absolutely right. Fixed.